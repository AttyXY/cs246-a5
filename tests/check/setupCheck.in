setup
+ K e1
+ k e5
+ r a1
done
quit 
quit
