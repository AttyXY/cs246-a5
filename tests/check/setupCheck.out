WELCOME TO CHESS! 
PLEASE ENTER ONE OF THE FOLLOWING COMMANDS TO BEGIN:
- setup
- game [player1] [player2]
- quit (to exit)

WELCOME TO SETUP MODE!
PLEASE ENTER ONE OF THE FOLLOWING COMMANDS TO CONTINUE:
- + [PieceType] [Coordinate] (to add piece)
- - [Coordinate] (to remove piece)
- = [Colour] (to set turn)
- done (to exit setup mode)

8  --------
7  --------
6  --------
5  --------
4  --------
3  --------
2  --------
1  ----K---
   abcdefgh

8  --------
7  --------
6  --------
5  ----k---
4  --------
3  --------
2  --------
1  ----K---
   abcdefgh

8  --------
7  --------
6  --------
5  ----k---
4  --------
3  --------
2  --------
1  r---K---
   abcdefgh

Invalid setup.

PLEASE ENTER THE FOLLOWING COMMANDS TO BEGIN:
- game [player1] [player2]

THANKS FOR PLAYING CHESS!
Final Score:
